ui/app/AppLayouts/Wallet/popups/swap/SwapInputParamsForm.qml

```diff
@@ -1,14 +1,55 @@
 import QtQml 2.15
 
+import StatusQ.Core.Utils 0.1 as SQUtils
+
 /* This is used so that there is an easy way to fill in the data
 needed to launch the Swap Modal with pre-filled requisites. */
 QtObject {
     id: root
+
+    signal formValuesChanged()
+
     property int selectedAccountIndex: 0
     property int selectedNetworkChainId: -1
     property string fromTokensKey: ""
     property string fromTokenAmount: "0"
     property string toTokenKey: ""
     property string toTokenAmount: "0"
     property double selectedSlippage: 0.5
+    property int lastEditedAmount: SwapInputParamsForm.LastEditedAmount.FromToken
+
+    onSelectedAccountIndexChanged: root.formValuesChanged()
+    onSelectedNetworkChainIdChanged: root.formValuesChanged()
```

Comment:
Not sure if done somewhere else or planned for other issue, but changing the network should reset the Swap modal to the initial state
https://www.figma.com/design/TS0eQX9dAZXqZtELiwKIoK/Swap---Milestone-1?node-id=3410-271102&t=uiWJXhqFKQoTertN-0
https://www.notion.so/Setup-Swap-59212ae7e94248b7aa26cafdd133e858?pvs=4#e64c1dfdfdaf4a1b88061b83fbb6685c

Reply:
HOW!! Did this change after my initial implementation pretty sure the reset of form wasn't there then . Thats not how its done currently. I can update this here https://github.com/status-im/status-desktop/pull/15161/files#:~:text=function%20test_modal_pay_input_switching_networks()%20%7B as Ive added a test for that in the other PR!